### ISSUE TITLE
In metaplex marketplace -- How is the marketplace owner making money ?  can someone please explain this ?

### ISSUE THREAD

**Issue by Harish88s** [metaplex-foundation/metaplex#606]
Hi All
I am wondering how is the owner of the marketplace making money ? what is the percentage or anyway to modify this percentage ?

Thank you

**Comment by BlopeetyBlop** [comment]
Can't find such answers on discord.

**Comment by Moikapy** [comment]
> Can't find such answers on discord.

I Agree, no one is answering these questions, can a dev help with this answer?

**Comment by BlopeetyBlop** [comment]
> 

they are not answering because at this current state metaplex is serving as single player store, e.g you can mint your own stuff, but nobody else can mint his stuff on your installation, regardless that it's advertised as NFT MARKETPLACE and all that. It should be marketed as 'personal nft store to mint your own nfts'

**Comment by Moikapy** [comment]
> > 
> 
> they are not answering because at this current state metaplex is serving as single player store, e.g you can mint your own stuff, but nobody else can mint his stuff on your installation, regardless that it's advertised as NFT MARKETPLACE and all that. It should be marketed as 'personal nft store to mint your own nfts'

I can have other accounts mint to my store although the process isn't automatic. Setting the site to the public allows users to mint, but doesn't automatically include/recognize their NFTs to be later auctioned off. I have to manually add their address to the site which defeats the purpose of me making the site public... I've been trying to find the component/code that handles purchasing and see if I can tweak it to include another transaction to pay a treasury wallet.
